http/src/request/channel/reaction/delete_all_reaction.rs

```diff
@@ -4,6 +4,7 @@ use twilight_model::{
     id::{ChannelId, MessageId},
 };
 
+/// Remove all reactions on a message of an emoji.
```

Comment:
This sounds a bit strange to me, maybe "Remove all reactions on a message for a given emoji" or maybe "Remove all of a specific emoji on a message" (Though again 2nd language)

Reply:
`Remove all reactions of a specified emoji from a message`